### ISSUE TITLE
suggested changes to Surgery form  + fix typo

### ISSUE THREAD

**Issue by tulste** [CoBrALab/ODC-Mouse-Forms#44]
type of Aneglesia; should be Analgesia

change order of questions + added questions needed:

1) Select physical intervention
2) Type of surgery (add intracerebral injection ;) )
3) was stereotax used?
if true, what stereotax
4) was anaesthesia used?
if true, what type and what dose
5) was analgesia used?
if true, what type and what dose
6) was hydration provided?
if true, what dose
7) length of surgery or start and end time
8) how long did it take for the mouse to recover?

**Comment by david-roper** [comment]
The form is to be done right after the surgery is completed so i would instead put expected recovery days for suggestion 8

**Comment by tulste** [comment]
no, not in terms of days, on the surgery card that get placed on the cage card, we need to record how long the mouse took to wake up from anaesthesia
